.github/workflows/studio-build.yml

```diff
@@ -0,0 +1,32 @@
+name: Studio Build
+
+on:
+  push:
```

Comment:
I don't think we need to do this on push Isaiah. Just doing this on PRs to master is fine since developers cant push to master directly anyway. Can you also add the studio branch here (in case we do hotfixes)